doc/dev-manual.md

```diff
@@ -99,4 +99,5 @@ Makefile rules:
 2. Then install `test-gninx` via `cpanm`:：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
 3. clone source code：`git clone https://github.com/openresty/test-nginx.git`;
 4. Load the `test-nginx` library with perl's `prove` command and run the test cases in the `/t` directory: `prove -Itest-nginx/lib -r t`
+>The environment variables TEST_NGINX_BINARY can set the execution path of nginx , such as TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t
```

Comment:
`The environment variables` --> `The environment variable`

Reply:
https://github.com/apache/incubator-apisix/pull/755/commits/eea2d0dde30c1c9c1bd821e0102d3fb847d5fa30